config/dev.exs

```diff
@@ -2,4 +2,5 @@ use Mix.Config
 
 config :vintage_net_wizard,
   backend: VintageNetWizard.Backend.Mock,
-  port: 4001
+  port: 4001,
+  captive_portal: false
```

Comment:
This seems like a legit change that we want for host-based testing. If you separate this out into a another PR, I'll merge right away. That way it won't be blocked by whatever is happening with the `phoenix_html` dependency.